lime/tools/helpers/IOSHelper.hx

```diff
@@ -319,144 +320,52 @@ class IOSHelper {
+			var currentDeviceID = XCodeHelper.getSimulatorId(project.targetFlags);
 			
 			try {
-				
-				ProcessHelper.runProcess ("", "open", [ "-Ra", "iOS Simulator" ], true, false);
-				ProcessHelper.runCommand ("", "open", [ "-a", "iOS Simulator", "--args", "-CurrentDeviceUDID", currentDeviceID ]);
-				
-			} catch (e:Dynamic) {
-				
-				ProcessHelper.runCommand ("", "open", [ "-a", "Simulator", "--args", "-CurrentDeviceUDID", currentDeviceID ]);
-				
+				ProcessHelper.runProcess("", "open", ["-Ra", "iOS Simulator"], true, false);
+				ProcessHelper.runCommand("", "open", ["-a", "iOS Simulator", "--args", "-CurrentDeviceUDID", currentDeviceID]);
+			} catch (e: Dynamic) {
+				ProcessHelper.runCommand("", "open", ["-a", "Simulator", "--args", "-CurrentDeviceUDID", currentDeviceID]);
 			}
 			
-			waitForDeviceState ("xcrun", [ "simctl", "uninstall", currentDeviceID, project.meta.packageName ]);
-			waitForDeviceState ("xcrun", [ "simctl", "install", currentDeviceID, applicationPath ]);
-			waitForDeviceState ("xcrun", [ "simctl", "launch", currentDeviceID, project.meta.packageName ]);
-			
-			ProcessHelper.runCommand ("", "tail", [ "-F", "~/Library/Logs/CoreSimulator/" + currentDeviceID + "/system.log"]);
+			waitForDeviceState("xcrun", ["simctl", "uninstall", currentDeviceID, project.meta.packageName]);
+			waitForDeviceState("xcrun", ["simctl", "install", currentDeviceID, applicationPath]);
+			waitForDeviceState("xcrun", ["simctl", "launch", currentDeviceID, project.meta.packageName]);
 			
+			ProcessHelper.runCommand("", "tail", ["-F", "~/Library/Logs/CoreSimulator/" + currentDeviceID + "/system.log"]);
 		} else {
-			
 			var applicationPath = "";
 			
-			if (Path.extension (workingDirectory) == "app" || Path.extension (workingDirectory) == "ipa") {
-				
+			if (Path.extension(workingDirectory) == "app" || Path.extension (workingDirectory) == "ipa") {
 				applicationPath = workingDirectory;
-				
 			} else {
-				
 				applicationPath = workingDirectory + "/build/" + configuration + "-iphoneos/" + project.app.file + ".app";
-				
 			}
 			
-			var templatePaths = [ PathHelper.combine (PathHelper.getHaxelib (new Haxelib ("lime")), "templates") ].concat (project.templatePaths);
-			var launcher = PathHelper.findTemplate (templatePaths, "bin/ios-deploy");
-			Sys.command ("chmod", [ "+x", launcher ]);
-			
-			var xcodeVersion = getXcodeVersion ();
-			
-			ProcessHelper.runCommand ("", launcher, [ "install", "--noninteractive", "--debug", "--bundle", FileSystem.fullPath (applicationPath) ]);
-			
+			var templatePaths = [PathHelper.combine(PathHelper.getHaxelib(new Haxelib("lime")), "templates")].concat(project.templatePaths);
+			var launcher = PathHelper.findTemplate(templatePaths, "bin/ios-deploy");
+			Sys.command("chmod", ["+x", launcher]);
+			// TODO: check if needed as not used
+			var xcodeVersion = getXcodeVersion();
```

Comment:
this prop is not used in the code in this branch safe to remove or it should be used?